pushapkscript/docker.d/worker.yml

```diff
@@ -20,7 +20,15 @@ products:
           skip_check_same_locales: true
           skip_checks_fennec: true
           apps:
-            # TODO: Rename this product `beta`
+            fenix-beta:
+              package_names: [ "org.mozilla.firefox_beta" ]
+              certificate_alias: 'fenix-beta'
+              google:
+                # TODO Revisit the default track once Fenix is kicked off by shipit
```

Comment:
🤔 I know this comment is just a copy-over, but I don't remember what it means 😅 For sure, Fenix has been kicked off by shipit for more than 2 years. As far as I know, there's noneed to change anything related to the Google Play tracks.

Thus, I'd recommend to remove this comment (and the same one down below) in order not to confuse future readers.

Reply:
Thanks. Done